File: TsogosunProfileAdmin/MSMDMPatron/Model/Dtos/GamingPointsTSGDto.cs

namespace tsogosun.com.MSMDMPatron.Model.Dtos
{
    public class GamingPointsTSGDto
    {
        public long? TsogosunID { get; set; }
        public decimal SlotPointsEarned { get; set; }
        public decimal TablePointsEarned { get; set; }
        public decimal TotalPointsEarned { get; set; }
        public int PointsToNextTier { get; set; }
        public decimal SlotPointsBalance { get; set; }
        public decimal TablePointsBalance { get; set; }
        public decimal? TotalPointsBalance { get; set; }
        public string FootNote_Line1 { get; set; }
        public string FootNote_Line2 { get; set; }
    }
}

File: TsogosunProfileAdminWeb/src/app/model/Dtos/mdm-patron/gamingPointsTSGDto.ts
export interface GamingPointsTSGDto {
    tsogosunID?: number | undefined;
    slotPointsEarned: number;
    tablePointsEarned: number;
    totalPointsEarned: number;
    pointsToNextTier: number;
    slotPointsBalance: number;
    tablePointsBalance: number;
    totalPointsBalance?: number | undefined;
    footNote_Line1?: string | undefined;
    footNote_Line2?: string | undefined;
}